External/Plugins/LintingHelper/Managers/LintingManager.cs

```diff
@@ -180,23 +179,38 @@ static void ApplyLint(string[] files, string language, List<LintingResult> resul
                         len -= result.FirstChar;
                     }
 
-                    var color = 0;
+                    var id = 0;
+                    int color = 0;
                     var lang = PluginBase.MainForm.SciConfig.GetLanguage(language);
                     switch (result.Severity)
                     {
                         case LintingSeverity.Error:
                             color = lang.editorstyle.ErrorLineBack;
+                            id = 3;
                             break;
                         case LintingSeverity.Warning:
                             color = lang.editorstyle.DebugLineBack;
+                            id = 4;
                             break;
                         case LintingSeverity.Info:
                             color = lang.editorstyle.HighlightWordBackColor;
+                            id = 5;
```

Comment:
Well, most numbers should work fine here, except 0, 1 and 2, because those are already used in various places:
https://github.com/fdorg/flashdevelop/blob/development/PluginCore/ScintillaNet/ScintillaControl.cs#L5719
https://github.com/fdorg/flashdevelop/blob/development/PluginCore/ScintillaNet/ScintillaControl.cs#L6800
https://github.com/fdorg/flashdevelop/blob/development/External/Plugins/AS3Context/Context.cs#L650
(to only name some of them)
But I think it would be better to have some api or something to handle this differently. From what I can tell, the only thing that counts, is that the indicators are different from each other. Otherwise they override each other.